kythe/go/indexer/indexer.go

```diff
@@ -44,21 +45,22 @@ import (
 	"go/types"
 	"io"
 	"io/ioutil"
+	"kythe.io/kythe/go/extractors/govname"
```

Comment:
These imports should be a separate group below the std lib imports and above the third_party ones.

Reply:
Done